python/hail/methods/statgen.py

```diff
@@ -126,3 +129,189 @@ def ld_matrix(dataset, force_local=False):
 
     jldm = Env.hail().methods.LDMatrix.apply(dataset._jvds, force_local)
     return LDMatrix(jldm)
+
+
+@handle_py4j
+@require_biallelic
+@typecheck(dataset=MatrixTable,
+           k=integral,
+           compute_loadings=bool,
+           as_array=bool)
+def hwe_normalized_pca(dataset, k=10, compute_loadings=False, as_array=False):
+    """Run principal component analysis (PCA) on the Hardy-Weinberg-normalized call matrix.
+
+    Examples
+    --------
+
+    >>> eigenvalues, scores, loadings = methods.hwe_normalized_pca(dataset, k=5)
```

Comment:
Slight inconsistency in docs here - normal PCA, you put loadings in `_`, but in this one (even though `compute_loadings=False` by default, it goes into an unused argument)

Reply:
I dont like having that option and am convinced that we can take it out. Will address in a future PR

Reply:
I was also wondering why it was there, but I assumed for speed?